extra/1-radio-stations.js

```diff
@@ -13,7 +13,10 @@
  * - Should return this array to use in other functions
  */
 
-// `getAllFrequencies` goes here
+function getAllFrequencies() {
+  let station = [87, 88, 89, 90, 91, 92, 93, 94, 95, 96, 97, 98, 99, 100, 101, 102, 103, 104, 105, 106, 107, 108]
```

Comment:
You can also write a for loop to create the array of frequecies.
```suggestion
  let frequencies = [];
  for (i = 87; i <= 108; i++) {
    frequencies.push(i);
  }
  return frequencies;
```

Reply:
That is ok. This is only a suggestion. Your function does the job and passes the test too.